yt_dlp/extractor/nbc.py

```diff
@@ -142,18 +145,40 @@ def _real_extract(self, url):
         theplatform_url = smuggle_url(update_url_query(
             'http://link.theplatform.com/s/NnzsPC/media/guid/%s/%s' % (video_data.get('mpxAccountId') or '2410887629', video_id),
             query), {'force_smil_url': True})
+
+        # fallbacks
+        description = video_data.get('description')
+        if description is None:
+            description = tpm.get('description')
+        episode_number = int_or_none(video_data.get('episodeNumber'))
+        if episode_number is None:
+            episode_number = int_or_none(tpm.get('nbcu$airOrder'))
+        rating = video_data.get('rating')
+        if rating is None:
+            try_get(tpm, lambda x: x['ratings'][0]['rating'])
+        season_number = int_or_none(video_data.get('seasonNumber'))
+        if season_number is None:
+            season_number = int_or_none(tpm.get('nbcu$seasonNumber'))
+        series = video_data.get('seriesShortTitle')
+        if series is None or series == '':
+            series = tpm.get('nbcu$seriesShortTitle')
+        tags = video_data.get('keywords')
+        if tags is None or len(tags) == 0:
+            tags = tpm.get('keywords')
```

Comment:
I figured you'd say that. Using `or` assumes there's only one fallback. Adding additional fallback cases in the future would result in nesting.

Reply:
Oh, the more important reason for my code: For most of the variables, `0` is a valid value, in which case the fallback should only be evaluated if the value is `None`. `rating` is sometimes `0`. `season_number` and `episode_number` are often `0` when a video is a bonus clip rather than an actual episode. In theory, series and description could be `0`.